src/mongocxx/test/spec/unified_tests/runner.cpp

```diff
@@ -445,6 +482,25 @@ options::client_session get_session_options(document::view object) {
     return session_opts;
 }
 
+options::client_encryption get_client_encryption_options(document::view object,
+                                                         client* key_vault_client) {
```

Comment:
```suggestion
options::client_encryption get_client_encryption_options(document::view, client* key_vault_client) {
```
Leave parameter unnamed instead of `(void)object`.